eng/Packages.Data.props

```diff
@@ -28,7 +28,7 @@
     <PackageReference Update="Azure.Storage.Common" Version="12.1.1" />
     <PackageReference Update="Azure.Storage.Files.Shares" Version="12.0.1" />
     <PackageReference Update="Azure.Storage.Queues" Version="12.1.1" />
-    <PackageReference Update="Azure.Storage.Files.DataLake" Version="12.0.0" />
+    <PackageReference Update="Azure.Storage.Files.DataLake" Version="12.5.0" />
```

Comment:
I did not find any other references to the package; this should not have a cascading impact.